tests/unit/stacked-vertical-bar-test.js

```diff
@@ -0,0 +1,773 @@
+import Ember from "ember";
+import { test, moduleForComponent } from 'ember-qunit';
+
+
+var three_ranges = [{
+   sliceLabel: "Label 1",
+   barLabel: "Group One",
+   value: 20
+}, {
+   sliceLabel: "Label 1",
+   barLabel: "Group Two",
+   value: 32
+}, {
+   sliceLabel: "Label 1",
+   barLabel: "Group Three",
+   value: 4
+}, {
+   sliceLabel: "Label 2",
+   barLabel: "Group One",
+   value: 16
+}, {
+   sliceLabel: "Label 2",
+   barLabel: "Group Two",
+   value: 17
+}, {
+   sliceLabel: "Label 2",
+   barLabel: "Group Three",
+   value: -18
+}, {
+   sliceLabel: "Label 3",
+   barLabel: "Group One",
+   value: -18
+}, {
+   sliceLabel: "Label 3",
+   barLabel: "Group Two",
+   value: 18
+}, {
+   sliceLabel: "Label 3",
+   barLabel: "Group Three",
+   value: -19
+}];
+
+// Adds additional slice types to the three_ranges data for higher granularity
+// and more thorough testing of slice sorting mechanics.
+var sliceSortingData = _.cloneDeep(three_ranges).concat([{
+  sliceLabel: "Label 4",
+  barLabel: "Group One",
+  value: -10
+}, {
+  sliceLabel: "Label 4",
+  barLabel: "Group Two",
+  value: -10
+}, {
+  sliceLabel: "Label 4",
+  barLabel: "Group Three",
+  value: 15
+}, {
+  sliceLabel: "Label 5",
+  barLabel: "Group One",
+  value: 10
+}, {
+  sliceLabel: "Label 5",
+  barLabel: "Group Three",
+  value: 10
+}, {
+  sliceLabel: "Label 6",
+  barLabel: "Group One",
+  value: 20
+}, {
+  sliceLabel: "Label 6",
+  barLabel: "Group Three",
+  value: 20
+}]);
+
+moduleForComponent('stacked-vertical-bar-chart', '[Unit] Stacked bar component', {
+  needs: [ 'template:components/chart-component'],
+  beforeEach: function () {},
+  afterEach: function () {}
+});
+
+test("it exists", function(assert) {
```

Comment:
Another artifact from VBC. Removing this -- if there's a problem with `this.subject()`, no other tests will pass anyway